### DISCUSSION TITLE
White background flashes on dark mode

### DISCUSSION THREAD

**Discussion by iamgiannisx** [razonyang/hugo-theme-bootstrap#667] [General]
I am not using the current version (alpha 1.0.7) yet but I am checking your website (https://hbs.razonyang.com/v1/en/) and when the page loads a white background flashes instanly and then the page turned into dark mode.

**Comment by razonyang** [comment]
Are you mean this issue which happens only on force refresh?
[Hugo-Theme-Bootstrap-Docs-v1.webm](https://user-images.githubusercontent.com/17720932/185755432-adb56c7c-6e02-4bda-95a9-8bc306a3398a.webm)

**Reply by iamgiannisx** [reply]
Exactly this is what I mean, it's not happening only at force refresh if you choose a different page e.g. tag, categories etc also happens.

**Reply by razonyang** [reply]
Yep, the issue does exist, the theme added a [blocking script](https://github.com/razonyang/hugo-theme-bootstrap/blob/master/assets/bootstrap/template.js) to avoid this, but it doesn't work very well.
I created an issue #668 for this, I haven't a solution at the moment, any thoughts are welcome.

**Comment by iamgiannisx** [comment]
I check through the inspector and you use [data-theme=dark] and [data-bs-theme=dark], I don't know if this is correct just sharing a thought.

**Comment by razonyang** [comment]
Thanks for your feedback, should be fixed by latest commit. You could preview it on https://hbs.razonyang.com/v1.
